gulp/tasks/test_debug.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        paths = {
+            spec: [
+                'test/specs/**/*.js'
+            ],
+            karamaConf: 'karma.debug.conf.js'
+        };
+
+    // starts up Karma using Chrome in debug mode
+    function runKarma(configFile, cb) {
```

Comment:
I believe `cb` is "callback" and I totally agree that it's unclear.  It's in the gulp documentation at https://github.com/gulpjs/gulp/blob/master/docs/API.md#accept-a-callback.